### DISCUSSION TITLE
Can we have an option to tell ray() to display its output in a fully expanded way?

### DISCUSSION THREAD

**Discussion by vesper8** [spatie/ray#375] [Ideas]
It would be nice to have an expand all / collapse all button

But it would also be nice to be able to do `ray($multiNestedArray)->expanded()`

**Comment by vesper8** [comment]
@freekmurze Pretty please?

`ray($largeNestedArray)->expanded()`

**Comment by AdrianMrn** [comment]
@vesper8  I think you can cmd+click or ctrl+click or shift+click (not sure which one 😄) on a symfony dump to recursively expand a node and all its children.

Let me know if this doesn't help you and I'll take a look if we can provide something!

**Comment by camiant** [comment]
+1

`->expanded()` would be very useful for reiterated logs on same complex object.. currently is quite annoying to have to "[X]+click" everytime a new log comes-in

**Comment by freekmurze** [comment]
It's coming soon: https://github.com/spatie/ray/pull/843 👀

**Reply by camiant** [reply]
you rock, "spatiers"! 😁🎉
